google/cloud/internal/unified_rest_credentials_test.cc

```diff
@@ -34,9 +41,33 @@ using ::google::cloud::MakeAccessTokenCredentials;
 using ::google::cloud::MakeGoogleDefaultCredentials;
 using ::google::cloud::MakeInsecureCredentials;
 using ::google::cloud::testing_util::IsOk;
+using ::google::cloud::testing_util::MakeMockHttpPayloadSuccess;
+using ::google::cloud::testing_util::MockRestClient;
+using ::google::cloud::testing_util::MockRestResponse;
 using ::google::cloud::testing_util::ScopedEnvironment;
+using ::google::cloud::testing_util::StatusIs;
+using ::testing::A;
+using ::testing::AtMost;
+using ::testing::ByMove;
+using ::testing::Contains;
 using ::testing::Eq;
+using ::testing::HasSubstr;
 using ::testing::IsEmpty;
+using ::testing::IsSupersetOf;
+using ::testing::MatcherCast;
+using ::testing::Pair;
+using ::testing::Property;
+using ::testing::Return;
+
+using MockClientFactory =
+    ::testing::MockFunction<std::unique_ptr<rest_internal::RestClient>(
+        Options const&)>;
+
+// Create a loadable, i.e., syntactically valid, key file, load it, and it
+// has the right contents.
```

Comment:
I'm not sure what this comment is referring to.

Reply:
Removed. Cut&paste error.